docs/source/reader_and_writer.rst

```diff
@@ -172,6 +172,39 @@ Basically, it's just a high level api on top of the native
         'rewriteBatchedStatements': 'true',  # improves write throughput dramatically
     })
 
+.. _redis:
```

Comment:
Sorry, missed the decision here: do we decided to have writer, and not having reader? + from the api it looks like you dictate that value is compressed json and how key is formatted - delimiter-separated. That is probably good defaults but I'd prefer to have it more generic. For example like in kafka: provide key-serializer and value-serializer as params, having your defaults, this will also make implementing reader much easier